cmd/ops_agent_windows/main_windows.go

```diff
@@ -108,7 +108,8 @@ func initServices() error {
 			filepath.Join(base, "google-cloud-metrics-agent_windows_amd64.exe"),
 			[]string{
 				"--config=" + filepath.Join(configOutDir, `otel\otel.yaml`),
-				"--feature-gates=exporter.googlecloud.OTLPDirect",
+				// TODO: remove postgres featuregate once emitWithResourceAttributes is enabled by default
+				"--feature-gates=exporter.googlecloud.OTLPDirect,receiver.postgresql.emitMetricsWithResourceAttributes",
```

Comment:
@qingling128 @ridwanmsharif 

So this feature gate is disabled by default will be enabled by default by the end of September hopefully in Otel. I was just double-checking with you guys on whether this was safe to add temporarily or will it cause issues down the line when the feature gate is removed?

Reply:
Tested an upgrade and figured I'd share what I've learned:

1. Service files stay in place even in upgrades.
2. Collector fails to start if a featuregate is specified that does not exist which in eventuality will become the case so this would eventually become an issue as users would have to modify their service file even if they don't use the postgresqlreceiver.

Reply:
Hmmmmm, it feels like to me that waiting for v0.60.0 is more reasonable. If we wanted to release this earlier than the september release date, we could potentially see if we can update the packaging/upgrade scripts to pick up the changes to the service unit files too. 

cc @qingling128 for opinions

Reply:
Do we have to remove the `receiver.postgresql.emitMetricsWithResourceAttributes` feature gate in Sept? Is it expected to be deprecated at that time? If so, +1 for waiting for `0.60.0`. If the `receiver.postgresql.emitMetricsWithResourceAttributes` feature gate is expected to be around for good, we can keep it and release now.

Reply:
@qingling128 

Yeah so for this case the featuregate is expected to be a temporary transition period that defaults to future behavior in september in the 0.60.0 release. Technically is deprecated now but default behavior at the current moment. The default behavior in 0.60.0 will be the future behavior and therefore doesn't need to specify the featuregate. 

I heard from @StefanKurek that @ridwanmsharif was going to look into fixing the issue with the ops agent not properly updating service info on upgrade. are you guys going to look into that while waiting for the v0.60.0 release

Reply:
Do we expect the v0.60.0 release to break some existing metrics for the Ops Agent? I'm wondering if upgrading to that version will take a while for us (i.e. if we have to deal with the breaking changes in the Ops Agent first).

If that is the case, we may need to prioritize the bug you identified now where the service unit files are not being updates with the upgrade.

Reply:
For postgres, v0.60.0 with the current modifications of flattening resource attributes, should be a purely additive change. Old dashboards/alerts should remain unaffected but new metrics should be added.

Reply:
This PR brings in essentially the v0.60.0 postgresqlreceiver https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/120